src/focus.service.ts

```diff
@@ -362,46 +332,35 @@ export class FocusService {
       return;
     }
 
-    this.rescroll(next, scrollSpeed, this.root);
-
     const attached = selected && isNodeAttached(selected, this.root);
 
-    this.parents = [];
-
     if (attached) {
       // Find the common ancestor of the next and currently selected element.
       // Remove selected classes in the current subtree, and add selected
       // classes in the other subtree. Trigger focus changes on every
       // element that we touch.
-      selected.blur();
-
       const common = getCommonAncestor(next, selected);
-      selected.classList.remove(cssClass.direct);
```

Comment:
for instance, if you remove arc--selected-direct and rely of focus, the player control buttons will start showing an outline if you click them on the website and those wont go away unless  you click anywhere else.